packages/reactivity/src/computed.ts

```diff
@@ -59,6 +60,12 @@ class ComputedRefImpl<T> {
   set value(newValue: T) {
     this._setter(newValue)
   }
+
+  cleanup() {
+    cleanup(this.effect)
+    this._dirty = true
```

Comment:
why is it necessary to set dirty here?

Reply:
By setting the dirty flag upon cleanup, when getting the computed value later, all dependencies will simply be recovered and the correct value will be returned. This way, it is possible to temporarily cleanup deps which in some cases can become a performance bottleneck.